src/VisualStudio/Core/Def/PickMembers/PickMembersDialog.xaml

```diff
@@ -155,12 +155,11 @@
                     <ItemsControl.ItemTemplate>
                         <DataTemplate>
                             <CheckBox IsChecked="{Binding IsChecked}"
-                                      VerticalAlignment="Center"
-                                      VerticalContentAlignment="Center"
-                                      Margin="8,0,0,5">
-                                <TextBlock Text="{Binding Title}"
-                                           TextWrapping="Wrap" />
-                            </CheckBox>
+                                VerticalAlignment="Center"
```

Comment:
This had to be done to correctly associate the keyboard press with the checkbox (and is slightly more correct anyways)